wandb-scim/users.py

```diff
@@ -200,22 +181,20 @@ def delete(self, user_id):
             str: A message indicating whether the user deletion was successful or failed.
         """
         print("Delete the User")
-        headers = {
-            "Authorization": self.authorization_header,
-            "Content-Type": "application/json",
-        }
         # Send a DELETE request to delete the user
         url = f"{self.base_url}/scim/Users/{user_id}"
-        response = requests.delete(url, headers=headers)
-
-        if response.status_code == 204:
-            return "User deleted successfully!"
-        elif response.status_code == 404:
-            return "User not found"
-        else:
+        try:
+            response = requests.delete(url, **self.request_kwargs)
+        except requests.HTTPError:
+            if response.status_code == 404:
+                return "User not found"
             return f"Failed to delete user. Status code: {response.status_code}"
+        return "User deleted successfully!"
```

Comment:
```suggestion
        return "User deletion succeeded"
```
I'm a little surprised we actually delete the deactivated user. I've often seen systems just leave them deactivated.

Reply:
same as above -- this preserves prior behavior